zerver/lib/digest.py

```diff
@@ -268,7 +272,21 @@ def gather_new_streams(
     return len(new_streams), {"html": channels_html, "plain": channels_plain}
 
 
-def enough_traffic(hot_conversations: str, new_streams: int) -> bool:
+def get_new_messages_count(user: UserProfile, threshold: datetime) -> int:
+    count = UserMessage.objects.filter(
+        user_profile=user, message__date_sent__gte=threshold, message__sender__is_bot=False
+    ).count()
+    return count
+
+
+def enough_traffic(
+    hot_conversations: str,
```

Comment:
This isn't a string ... and hasn't ever been a string as far as I can tell. Probably worth a prep commit to fix that up.